### ISSUE TITLE
Location Field

### ISSUE THREAD

**Issue by ahinerfeld** [CalderaWP/Caldera-Forms#2975]
I would like to see the ability to add a location field that includes Google Maps or Open Street Maps so that you can mark your position on a map.  The field should have an auto locate option that allows you to use your IP address for geolocation or if on a mobile device your GPS location.

**Comment by New0** [comment]
Hello @ahinerfeld , thank you for sharing the feature request. This could be developed as an add-on.

Did you know the [Google Maps geolocation add-on](https://calderaforms.com/downloads/geolocation-field-for-caldera-forms/) allows geolocation based on an address input ?

**Comment by ahinerfeld** [comment]
Yes I have looked at this but I am not at an address I am out in the middle of a National Park and I am trying to identify where a tree has fallen or a bridge is out.  Do you know if this plugin will allow for "Current Location" to pull in an address or GPS location?

**Comment by ahinerfeld** [comment]
Just checking in on this request.

**Comment by ahinerfeld** [comment]
Any chance of seeing an add-on to do this?
